.github/workflows/security.yml

```
@@ -12,8 +12,55 @@ on:
+          path: packages/core/coverage.xml
+
+      - name: Upload coverage HTML report
+        uses: actions/upload-artifact@v4
+        with:
+          name: coverage-html
+          path: packages/core/htmlcov/
+
   python-security:
+    needs: tests
     runs-on: ubuntu-latest
+    needs: tests
```

actionlint (1.7.9)

key "needs" is duplicated in "python-security" job. previously defined at line:61,col:5

(syntax-check)